blockers.go

```diff
@@ -29,6 +36,37 @@ type ItemInfo struct {
 	Status      string `json:"status"`
 }
 
+func fetchExpressBouncerDownload() (int, error) {
+	now := time.Now()
+	nowFmt := fmt.Sprintf("%d-%d-%d", now.Year(), now.Month(), now.Day())
+	bouncerFirstReleaseDate := "2021-01-01"
+	url := fmt.Sprintf("https://api.npmjs.org/downloads/point/%s:%s/@crowdsec/express-bouncer", bouncerFirstReleaseDate, nowFmt)
```

Comment:
According to the [documentation](https://github.com/npm/registry/blob/master/docs/download-counts.md#limits) : `All other queries are limited to at most 18 months of data.`
Add a check to make sure we don't request more than 18 months at a time, and if so, split the time range so that we can get all the downloads ?